spec/json/matchers/match_response_schema_spec.rb

```diff
@@ -24,25 +24,26 @@
 
   it "fails when the body contains a property with the wrong type" do
     create_schema("foo_schema", {
-      type: "object",
-      properties: {
-        foo: { type: "string" }
+      "type" => "object",
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.